proto/message_contents/public_key.proto

```diff
@@ -9,6 +9,53 @@ import "message_contents/signature.proto";
 option go_package = "github.com/xmtp/proto/v3/go/message_contents";
 option java_package = "org.xmtp.proto.message.contents";
 
+// An unsigned public key used by libxmtp
+message VoodooUnsignedPublicKey {
+    uint64 created_ns = 1;
+
+    oneof union {
+        VodozemacCurve25519 curve25519 = 2;
+    }
+
+    // A Vodozemac curve25519 key serialized via serde
+    message VodozemacCurve25519 {
```

Comment:
This comment is tough because "via serde" doesn't describe the actual serialization/deserialization format. You can link to the exact Rust class + commit and that should provide enough context e.g. link to vodozemac Github at commit X